hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/PrefetchExecutor.java

```diff
@@ -53,7 +56,8 @@ public final class PrefetchExecutor {
   private static final Map<Path, Future<?>> prefetchFutures = new ConcurrentSkipListMap<>();
   /** Set of files for which prefetch is completed */
   @edu.umd.cs.findbugs.annotations.SuppressWarnings(value = "MS_SHOULD_BE_FINAL")
-  private static HashMap<String, Boolean> prefetchCompleted = new HashMap<>();
+  private static ConcurrentHashMap<String, Long> regionPrefetchSizeMap = new ConcurrentHashMap<>();
```

Comment:
OOP good practices: "Program to interfaces"`Map<String, Long> regionPrefetchSizeMap = new ConcurrentHashMap<>();`

Reply:
Done.